variables.tf

```diff
@@ -1,5 +1,78 @@
-#variable "example" {
-#  type        = string
-#  default     = ""
-#  description = "This is just an example of a variable"
-#}
+variable "image_url" {
+  type        = string
+  default     = "public.ecr.aws/p5r4i7k7/sidekick:latest"
+  description = "The container image url for Lacework sidekick."
+}
+variable "resource_name_prefix" {
+  type        = string
+  description = "A string to be prefixed to the name of all new resources."
+}
+variable "resource_name_suffix" {
+  type        = string
+  description = "A string to be appended to the end of the name of all new resources."
+}
+
+variable "lacework_integration_name" {
+  type        = string
+  description = "The name of the Lacework cloud account integration."
+  default     = "aws-agentless-scan"
+}
+
+variable "scan_frequency_hours" {
+  type        = number
+  description = "How often in hours the scan will run in hours. Defaults to `24`."
+  default     = 24
+}
+
+variable "filter_query_text" {
+  type        = string
+  description = "The LQL query text."
+  default     = ""
+}
+
+variable "scan_containers" {
+  type        = bool
+  description = "Whether to includes scanning for containers."
+  default     = false
+}
+variable "scan_host_vulnerabilities" {
+  type        = bool
+  description = "Whether to includes scanning for host vulnerabilities."
+  default     = false
+}
```

Comment:
Both of these options default to `true` in the UI, and I suspect most customers will want them both enabled.  

```suggestion
variable "scan_containers" {
  type        = bool
  description = "Whether to includes scanning for containers."
  default     = true
}
variable "scan_host_vulnerabilities" {
  type        = bool
  description = "Whether to includes scanning for host vulnerabilities."
  default     = true
}
```